docs/README.md

```diff
@@ -105,77 +107,43 @@ environment variables as parameters:
-CORECLR_PROFILER_PATH_64
-OTEL_DOTNET_AUTO_INTEGRATIONS_FILE
-```
-
-You can also use the [instrument.sh](../instrument.sh) script which uses following
-environment variables as parameters:
+| Environment variable                 | For                    | Value                                                                               |
+|--------------------------------------|------------------------|-------------------------------------------------------------------------------------|
+| `COR_ENABLE_PROFILING`               | .NET Framework         | `1`                                                                                 |
+| `COR_PROFILER`                       | .NET Framework         | `{918728DD-259F-4A6A-AC2B-B85E1B658318}`                                            |
+| `COR_PROFILER_PATH_32`               | .NET Framework         | `$INSTALL_DIR/win-x86/OpenTelemetry.AutoInstrumentation.Native.dll`                 |
+| `COR_PROFILER_PATH_64`               | .NET Framework         | `$INSTALL_DIR/win-x64/OpenTelemetry.AutoInstrumentation.Native.dll`                 |
+| `CORECLR_ENABLE_PROFILING`           | .NET (Core)            | `1`                                                                                 |
+| `CORECLR_PROFILER`                   | .NET (Core)            | `{918728DD-259F-4A6A-AC2B-B85E1B658318}`                                            |
+| `CORECLR_PROFILER_PATH`              | .NET (Core) on Linux   | `CORECLR_PROFILER_PATH=$INSTALL_DIR/OpenTelemetry.AutoInstrumentation.Native.so`    |
+| `CORECLR_PROFILER_PATH`              | .NET (Core) on macOS   | `CORECLR_PROFILER_PATH=$INSTALL_DIR/OpenTelemetry.AutoInstrumentation.Native.dylib` |
+| `CORECLR_PROFILER_PATH_32`           | .NET (Core) on Windows | `$INSTALL_DIR/win-x86/OpenTelemetry.AutoInstrumentation.Native.dll`                 |
+| `CORECLR_PROFILER_PATH_64`           | .NET (Core) on Windows | `$INSTALL_DIR/win-x64/OpenTelemetry.AutoInstrumentation.Native.dll`                 |
+| `DOTNET_ADDITIONAL_DEPS`             | .NET (Core)            | `$INSTALL_DIR/AdditionalDeps`                                                       |
+| `DOTNET_SHARED_STORE`                | .NET (Core)            | `$INSTALL_DIR/store`                                                                |
+| `DOTNET_STARTUP_HOOKS`               | .NET (Core)            | `$INSTALL_DIR/netcoreapp3.1/OpenTelemetry.AutoInstrumentation.StartupHook.dll`      |
+| `OTEL_DOTNET_AUTO_HOME`              | all                    | `$INSTALL_DIR`                                                                      |
+| `OTEL_DOTNET_AUTO_INTEGRATIONS_FILE` | all                    | `$INSTALL_DIR/integrations.json`                                                    |
+
+> Some configuration can be omitted on .NET (Core). More info [here](config.md#net-clr-profiler).
+
+Additionally, set the [resources](config.md#resources) when running your application.
```

Comment:
@pellared Can we make this a step previous to the settings table? 

1. Install the instrumentation
2. Define mandatory env vars and resources
3. Further configure

Reply:
I will probably need a little help. Maybe you will be able to create a little PR or we will try to call each other 😉

Reply:
I tried here 8e9a44f99